src/main/resources/log4j.properties

```diff
@@ -0,0 +1,16 @@
+# Root logger option
+log4j.rootLogger=INFO, stderr, logfile
+
+# Direct log messages to stderr
+log4j.appender.stderr=org.apache.log4j.ConsoleAppender
+log4j.appender.stderr.Target=System.err
+log4j.appender.stderr.layout=org.apache.log4j.PatternLayout
+log4j.appender.stderr.threshold=WARN
+log4j.appender.stderr.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
+
+# Log at INFO level to file
+log4j.appender.logfile=org.apache.log4j.FileAppender
```

Comment:
does this mean INFO will be output into a file somewhere?

I've started seeing this message, which seems like it's failing to find `logfile` and falling back to outputting info to the screen.  How is this intended to interact with our verbosity settings?

```
log4j:WARN File option not set for appender [logfile].
log4j:WARN Are you using FileAppender instead of ConsoleAppender?
log4j:ERROR No output stream or file set for the appender named [logfile].
```